doc/wg/core/notes/core-notes-2025-12-04.md

```diff
@@ -0,0 +1,48 @@
+# Tock Meeting Notes 2025-12-04
+
+## Attendees
+ - Branden Ghena
+ - Hudson Ayers
+ - Johnathan Van Why
+ - Leon Schuermann
+ - Amit Levy
+
+
+## Updates
+### IPC
+ * Branden: https://github.com/tock/tock/pull/4680
+ * Branden: This is an RFC documenting IPC efforts. It mostly aligns with our discussions from September at Tockworld, actually recording them in writing. It also includes goals/non-goals and an example of how mechanisms could be used for a Thread service.
+ * Branden: This documentation does NOT include any shared memory mechanism yet. We wanted to get this out for eyes first, then work on an implementation for allow-based messaging, and then circle back to shared memory which is going to be more complicated.
+ * Branden: For now, our ask is just eyes on this when you have time. If you spot anything that seems problematic or that you disagree with, let us know.
+### Removing Static Mut
+ * Amit: Working on abstraction for panic UART with single-thread value. In the works still
+ * Amit: Works for one board, but has conflicts and needs to be added to other boards
+ * Leon: It would be good to get reviews on a draft for a single board first
+
+
+## Single Thread Value
+ * Leon: New panic resource for debug. A few calls ago we had a discussion about removing static muts. We established a mini taskforce of Brad, Amit, and Leon to figure out how to fix it. Brad has put in a lot of effort on this. One long-standing issue is how invasive and non-mechanical these efforts are.
+ * Leon: So to move the efforts forward, what I think Brad intends is to get more eyes on this new PR https://github.com/tock/tock/pull/4676
+ * Leon: This PR take the non-mechanical static muts for resources shared between board and panic routines in io.rs and stuffs them into a struct, which is itself within a Single-Thread Value. This PR ports all boards to use this struct-based approach. It doesn't remove all static muts, notably UARTs for printing and some for tests still exist. But if this was merged, it would be a massive step forward to getting rid of the static muts.
```

Comment:
```suggestion
 * Leon: This PR take the mechanical static muts for resources shared between board and panic routines in io.rs and stuffs them into a struct, which is itself within a Single-Thread Value. This PR ports all boards to use this struct-based approach. It doesn't remove all static muts, notably UARTs for printing and some for tests still exist. But if this was merged, it would be a massive step forward to getting rid of the static muts.
```

Probably misspoke there.